src/app/components/Icons/HomeIcon.tsx

```diff
@@ -0,0 +1,12 @@
+import SVG, { SVGProps } from '../IconSvg/IconSvg';
+
+function HomeIcon(props: Omit<SVGProps, 'children'>): JSX.Element {
+  return (
+    <SVG {...props}>
+      <path d="M0 0h24v24H0z" fill="none" />
```

Comment:
Maybe not required too

Reply:
we could get rid of it, but for the specific design its necessary